app/src/components/validators.js

```diff
@@ -506,6 +506,15 @@ const validatorUtils = {
   /** @function isString */
   isString: x => {
     return Object.prototype.toString.call(x) === '[object String]';
+  },
+
+  /** @function blockDoNotReplySender */
+  blockDoNotReplySender: x => {
+    if (config.has('server.blockDoNotReplySender')) {
+      return x.match(/^(do)?no(t|-)?reply/gi);
```

Comment:
After team discussion, I think we're rolling back to only enforcing a block on the exact donotreply address, not the expanded scope one.